docs/installation/generic.md

```diff
@@ -106,3 +108,112 @@ gunicorn -b 0.0.0.0:8080 app:app --enable-stdio-inheritance --log-level "info"
 ```
 
 You can now visit [http://localhost:8080](http://localhost:8080/) on your browser to access the app generator.
+ ### 2. OSX:
+
+
+### Cloning the Project
+```bash
+git clone https://github.com/fossasia/open-event-android.git && cd open-event-android
+export PROJECT_DIR=$(pwd)
+```
+
+### Installing the requirements
+
+#### Installing dependencies
+
+### Install brew
+```bash
+ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+  xcode-select --install
+```
+```bash
+brew install software-properties-common git wget libc6-i386 lib32stdc++6 lib32gcc1 lib32ncurses5 lib32z1 curl libqt5widgets5
+brew install ca-certificates && update-ca-certificates
+```
+
+#### Installing Python 2.7.x
+
+```bash
+brew install python 
+brew install python-dev 
+brew install python-pip 
+brew install libpq-dev 
+brew install libevent-dev 
+brew install libmagic-dev 
+```
+
+#### Installing Oracle JDK 8
+```bash
+You can follow this link (https://docs.oracle.com/javase/8/docs/technotes/guides/install/mac_jdk.html) to install Oracle JDK on your machine.
+```
+
+#### Installing Android SDK
```

Comment:
Here in installation process all other steps are same except installing brew and python.
So it will be good to remove redundant documentation.